### DISCUSSION TITLE
en/lc/2161/

### DISCUSSION THREAD

**Discussion by giscus** [doocs/leetcode#3609] [Announcements]
# en/lc/2161/

LeetCode solutions in any programming language

https://leetcode.doocs.org/en/lc/2161/

**Comment by sishir2001** [comment]
Isn't there any algo that does it in in-place . T(N) = O(N) and S(N) = O(1) ?

**Reply by sishir2001** [reply]
@yanglbme

**Comment by AbelTarazona** [comment]
Nicee
